node_package/tests/ReactOnRails.test.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable react/no-multi-comp */
+import test from 'tape';
+import React from 'react';
+import ReactOnRails from '../src/ReactOnRails';
+import { canUseDOM } from 'fbjs/lib/ExecutionEnvironment';
+import JsDom from 'jsdom';
+
+if (!canUseDOM) {
+  global.document = JsDom.jsdom('<div id="root"></div>');
+  global.window = document.defaultView;
+}
+
+test('ReactOnRails render returns React element', (assert) => {
+  assert.plan(1);
+  const R1 = React.createClass({
+    render() {
+      return (
+        <div> WORLD </div>
+      );
+    },
+  });
+  ReactOnRails.register({ R1 });
+  const actual = ReactOnRails.render('R1', {}, 'root')._reactInternalInstance._currentElement.type;
+  assert.deepEqual(actual, R1,
```

Comment:
are we comparing elements or components?

https://facebook.github.io/react/docs/glossary.html